libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -223,30 +223,30 @@ class ServiceProviderFactoryClient extends GovernedContractClient {
     return info
   }
 
-  async getServiceProviderIdsFromAddress (ownerAddress, serviceType) {
+  async getServiceProviderIdsFromAddress(ownerAddress, serviceType) {
     const method = await this.getMethod('getServiceProviderIdsFromAddress',
       ownerAddress,
       Utils.utf8ToHex(serviceType)
     )
     let info = await method.call()
-    return info
+    return info.map(id => parseInt(id))
   }
 
-  async getServiceProviderIdFromAddress (ownerAddress, serviceType) {
+  async getServiceProviderIdFromAddress(ownerAddress, serviceType) {
     const infos = await this.getServiceProviderIdsFromAddress(ownerAddress, serviceType)
-    return infos[0]
+    return infos[0] ? parseInt(infos[0]) : null
   }
 
-  async getServiceEndpointInfoFromAddress (ownerAddress, serviceType) {
+  async getServiceEndpointInfoFromAddress(ownerAddress, serviceType) {
```

Comment:
similar comment here, this doesn't really work since we have >1 endpoint per address - we can get a list of IDs and the endpoints but this function as is can't be guaranteed to be right

Reply:
defering to @raymondjacobson

Reply:
Let's keep this around until we fully deprecate the old dashboard.

Reply:
makes sense!